app/auth/gitlab_auth.py

```diff
@@ -238,5 +239,8 @@ def get_gitlab_refresh_token(access_token):
 @blueprint.route('/logout')
 def logout():
     logout_url = current_app.config['GITLAB_URL'] + '/users/sign_out'
-    response = current_app.make_response(redirect(logout_url))
-    return response
+    return render_template(
```

Comment:
instead of making this breaking change, could we have a configuration variable that allows us to satisfy gitlab pre and post `12.9`?

Reply:
I generally like the idea, but it's more annoying than it seems. Currently, the gateway is not aware of the gitlab version. The gitlab API has a `/version` endpoint, but only available for logged-in users, so it can't just be queried at startup. Doing this in a clean way would therefore need non-negligible chart refactoring. I agree that we will eventually have to support multiple versions of the same dependency, as long as we can avoid this work by upgrading RenkuLab and GitLab in the SDSC deployments I would prefer avoiding this a little longer.

Reply:
why does it need to be automatic? It can be a chart value that whoever is deploying the platform can set.

Reply:
done as discussed